<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="label_pic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>301</width>
        <height>321</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>411</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Sing in</string>
      </property>
      <widget class="QPushButton" name="pushButton_login">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>230</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>361</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>UserName  :    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_username">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>361</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Password    :    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_of_choose">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>421</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="virtualMapButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Use virtual map</string>
       </property>
      </widget>
      <widget class="QPushButton" name="realMapButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>80</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Use real map</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="singOutButton_1">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>20</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign out</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Abyssinica SIL</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose the type of map</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_of_RealMap">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>581</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Postal Address</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lePostalAddress"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Zoom</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="zoomSpinBox">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="goButton">
         <property name="text">
          <string>Go</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="pbRemoveMarker">
         <property name="text">
          <string>Remove marker</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>581</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWebView" name="webView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>561</width>
         <height>411</height>
        </rect>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>390</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Vehicle information</string>
      </property>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Top Size</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>44</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Top Weight</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>98</y>
        <width>191</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Add product</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>96</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>  Weight</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>132</y>
         <width>55</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>   Size      </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>84</x>
         <y>94</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>83</x>
         <y>130</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>240</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>      Id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>84</x>
         <y>58</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Lc</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select Location</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="signOutButton_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_of_VirtualMap">
     <widget class="QPushButton" name="signOutButton_3">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>20</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>20</y>
        <width>631</width>
        <height>551</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>611</width>
         <height>511</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="viewContentLayout"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>90</y>
        <width>151</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="weight">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>110</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="size">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>170</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="setButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>86</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Weight</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>150</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>105</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <widget class="QLabel" name="Idlabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>     ...     </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Id  :  </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="selected">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>     ...</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>400</y>
        <width>151</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Coût total du cycle
       hamiltonien:</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="labelResut">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Size
Weight</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>79</x>
         <y>90</y>
         <width>61</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="sizeWeight">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>26</y>
          <width>41</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Number of vertex</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nVertex">
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhFormattedNumbersOnly</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="generateButton">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>22</width>
        <height>58</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="zoomIn">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="zoomOut">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>340</y>
        <width>151</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="BAndBButton">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Calculate B and B</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="calculTSPButton">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Calculate TSP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
